deploy/blobfuse2/config.yaml

```diff
@@ -0,0 +1,23 @@
+allow-other: true
+
+logging:
+  type: base
+  level: log_debug
+
+components:
+  - libfuse
+  - file_cache
+  - attr_cache
+  - azstorage
+
+libfuse:
+  attribute-expiration-sec: 120
+  entry-expiration-sec: 120
+  negative-entry-expiration-sec: 240
+
+file_cache:
+  timeout-sec: 120
+  max-size-mb: 4096
+
+attr_cache:
+  timeout-sec: 7200
```

Comment:
Do we need to give azstorage config where user gives storage account details, or those have been ignored explicitly.

Reply:
We support all the env variables that v1 supports so if that is the scenario for auth the customer uses with CSI driver it should be fine.